### DISCUSSION TITLE
OTP on COD

### DISCUSSION THREAD

**Discussion by itspathanofficial** [inspireui/support#5594] [New Feature/Ideas Requests]
Some people create accounts with fake Gmail or Facebook login in the app and place an order with a fake address just to disturb the company, other competitor companies also do the same.

We are facing this issue for a long time which is why we have turned off our COD option.

Inspireui is it’s is really innovative so I have a feature suggestion on a request that adds an OTP option on COD orders, whenever someone orders with COD they have to verify their contact number, almost all reputed e-commerce companies do the same in India.

**Comment by krrisharora** [comment]
Yes that would is a required feature. Even many people enter wrong phone number in address or sometime they enter pincode or email address in the phone number. So we need this feature asap.

Also you have mentioned: @itspathanofficial  "**We are facing this issue for a long time which is why we have turned off our COD option.**" I  Just want to ask how did you manage to turn off COD in the app, because there is no such option for this in the app.

Thanks

**Reply by itspathanofficial** [reply]
We are using WordPress for our business as of now so it is so much easy to turn it off, all you have to do is go to WooCommerce->Settings->Payments and disable "Cash on delivery".
